src/Instrument/Transformer/CachingTransformer.php

```diff
@@ -99,6 +109,9 @@ public function transform(StreamMetaData $metadata)
             }
             $this->processTransformers($metadata);
             file_put_contents($cacheUri, $metadata->source);
+            if (!$cacheExisted && $this->cachePerms){
+                chmod($cacheUri, $this->cachePerms);
```

Comment:
It isn't always possible to change the permissions of file that existed before file_put_contents call, since one must be the owner of the file to do so.

If we have different parts of the app running under different users, we'll get a permission problems here.